lib/repl.js

```diff
@@ -58,6 +58,7 @@ const Module = require('module');
 const domain = require('domain');
 const debug = util.debuglog('repl');
 const errors = require('internal/errors');
+const inspector = require('inspector');
```

Comment:
I guess this might be need to be guarded with something for `--without-inspector`?

Reply:
done